pyi.py

```diff
@@ -442,6 +500,52 @@ def _visit_function(self, node: ast.FunctionDef | ast.AsyncFunctionDef) -> None:
                     continue
             self.error(statement, Y010)
 
+        # Rest of this method is devoted to finding TypeVars that should be _typeshed.Self, but aren't
```

Comment:
This comment seems to be stating the obvious. Am I missing something?

Reply:
Oh sorry, I changed it and didn't delete the comment.

Reply:
Since this function is getting pretty big, it may be worth adding separate functions for the individual areas we check.